mintpy/smallbaselineApp.py

```diff
@@ -208,6 +208,8 @@ class TimeSeriesAnalysis:
     def __init__(self, customTemplateFile=None, workDir=None):
         self.customTemplateFile = customTemplateFile
         self.workDir = os.path.abspath(workDir)
+        if not 'mintpy' in self.workDir:
+            self.workDir = self.workDir + '/mintpy'
```

Comment:
To have `mintpy` in the working directory, one can user `--dir` option to specify that manually. This change would force the user to have `mintpy` in their directory path. I thought you would not like it, considering combing with minopy workflow. Are you sure about this?

Reply:
If it is not there, it produces error while I am in project directory. but you are right with --dir there is no problem. I leave it to you to decide
No problem with minopy

Reply:
Let's keep the old way then, I think it's more consistent on the user side.